docs/configuration/index.md

```diff
@@ -218,7 +218,7 @@ Example usage:
 
 When unset, default release suffix that is generated by Packit is following:
 
-    {original_release_number}.{current_time}.{sanitized_current_branch}{git_desc_suffix}
+    {original_release_number}.{current_time}.{sanitized_current_branch}.{git_desc_suffix}
```

Comment:
It appears like that, but it's a consequence of the `git_desc_suffix` being prefixed with the `.`, so in the template, the dot is actually not present…

Reply:
Aaah!